airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/AbstractSourceConnectorTest.java

```diff
@@ -133,6 +133,10 @@ protected ConnectorSpecification runSpec() throws WorkerException {
   }
 
   protected StandardCheckConnectionOutput runCheck() throws Exception {
+    return runCheck(getConfig());
+  }
+
+  protected StandardCheckConnectionOutput runCheck(JsonNode config) throws Exception {
```

Comment:
How is `JsonNode config` used here?

Reply:
Sorry, but these are not my changes, but github for some reason shows that they are mine
![2022-08-26_11-52](https://user-images.githubusercontent.com/88329385/186866536-d73e78f7-e8c6-4aab-badc-c38ed3ad3b3d.png)
.

Reply:
Well noted. Now I no longer use this logic. Deleted it.